doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -189,6 +189,26 @@ KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
 Bluetooth LE
 ============
 
+.. rst-class:: v2-9-0-nRF54H20-rc1 v2-9-0 v2-8-0
+
+NCSDK-31528: Blocking the system workqueue if the :kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL` Kconfig option is disabled can cause a deadlock in the Bluetooth Host when running out of buffers in the HCI commands pool.
```

Comment:
Sentence parsing error. Something went wrong in the rephrasing and now it combination of two sentences with two verbs into single one.

Perhaps?
```suggestion
NCSDK-31528: Issue with blocking of the system workqueue. If the :kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL` Kconfig option is disabled can cause a deadlock in the Bluetooth Host when running out of buffers in the HCI commands pool.
```

Reply:
The first line with the Jira issue ID is usually the same as the title in the issue, without a fullstop. Therefore, suggesting:
NCSDK-31528: Deadlock on system workqueue with `tx_notify` in host
  If the :kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL` Kconfig option is disabled, blocking of the system workqueue can cause a deadlock in the Bluetooth Host when running out of buffers in the HCI commands pool.

Reply:
Second line must be indented a couple of spaces (does not seem to show it properly in my comment.